notebooks/load.ipynb

```diff
@@ -0,0 +1,426 @@
+    "                      'velocities': [[9883868.06162645, 8989183.29614645, 881.9339912834714]],\n",
+    "                      'unit': 'm'},\n",
+    "     'sensor_orientation': {'quaternions': [[0.0839325155418465, 0.01773153459973076, 0.9946048838768001, 0.05832709905329919],\n",
+    "                                            [0.08400255389846112, 0.017728573660888338, 0.9945982552324419, 0.058340203145592955],\n",
+    "                                            [0.0840727438677933, 0.01772597983213093, 0.9945916990701015, 0.058351653947255284],\n",
+    "                                            [0.08414295024563542, 0.0177232314764712, 0.9945850926215932, 0.058363897444302155],\n",
+    "                                            [0.08421298603777543, 0.017720474751607065, 0.9945784602692478, 0.05837674302007016],\n",
+    "                                            [0.08428244063999622, 0.01771827039990229, 0.9945719105583338, 0.058388764519787986]]},\n",
+    "     'detector_sample_summing': 1,\n",
+    "     'detector_line_summing': 1,\n",
+    "     'focal_length_model': {'focal_length': 352.9271664},\n",
+    "     'detector_center': {'line': 0.430442527, 'sample': 2542.96099},\n",
+    "     'starting_detector_line': 0,\n",
+    "     'starting_detector_sample': 0,\n",
+    "     'focal2pixel_lines': [0.0, 142.85714285714, 0.0],\n",
+    "     'focal2pixel_samples': [0.0, 0.0, 142.85714285714],\n",
+    "     'optical_distortion': {'radial': {'coefficients': [-0.0073433925920054505, 2.8375878636241697e-05, 1.2841989124027099e-08]}},\n",
+    "     'image_lines': 400,\n",
+    "     'image_samples': 5056,\n",
+    "     'name_platform': 'MARS_RECONNAISSANCE_ORBITER',\n",
+    "     'name_sensor': 'CONTEXT CAMERA',\n",
+    "     'reference_height': {'maxheight': 1000, 'minheight': -1000, 'unit': 'm'},\n",
+    "     'name_model': 'USGS_ASTRO_LINE_SCANNER_SENSOR_MODEL',\n",
+    "     'interpolation_method': 'lagrange',\n",
+    "     'line_scan_rate': [[0.5, -0.37540000677108765, 0.001877]],\n",
+    "     'starting_ephemeris_time': 297088762.24158406,\n",
+    "     'center_ephemeris_time': 297088762.61698407,\n",
+    "     't0_ephemeris': -0.37540000677108765,\n",
+    "     'dt_ephemeris': 0.15016000270843505,\n",
+    "     't0_quaternion': -0.37540000677108765,\n",
+    "     'dt_quaternion': 0.15016000270843505}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.1"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
What's up with this notebook?

Reply:
same as vis, was supposed to stay local.